Northstar.CustomServers/mod/scripts/vscripts/evac/_evac.gnut

```diff
@@ -428,3 +494,89 @@ void function EvacDropshipKilled( entity dropship, var damageInfo )
 		}
 	}
 }
+
+// if there's no player left in evacing team, we end this match
+void function CheckIfAnyPlayerLeft( int evacTeam )
+{
+	wait GAME_EPILOGUE_PLAYER_RESPAWN_LEEWAY
+	float startTime = Time()
+
+	OnThreadEnd(
+		function() : ( evacTeam )
+		{
+			SetTeamActiveObjective( evacTeam, "EG_DropshipExtractEvacPlayersKilled" )
+			SetTeamActiveObjective( GetOtherTeam( evacTeam ), "EG_StopExtractEvacPlayersKilled" )
+			thread EvacEpilogueCompleted( null )
+		}
+	)
+	while( true )
+	{
+		if( GetPlayerArrayOfTeam_Alive( evacTeam ).len() == 0 )
+			break
+		if( GetGameState() == eGameState.Postmatch )
+			return
+		WaitFrame()
+	}	
+}
+
+void function DropShipTempHide( entity dropship )
+{
+	dropship.kv.VisibilityFlags = 0 // or it will still shows the jetpack fxs
+	HideName( dropship )
+	wait 0.46
+	if( IsValid( dropship ) )
+	{
+		dropship.kv.VisibilityFlags = ENTITY_VISIBLE_TO_EVERYONE
+		ShowName( dropship )
+	}
+}
+
+EvacShipSetting function GetEvacShipSettingByTeam( int team )
+{
+	EvacShipSetting tempSetting
+	if( team == TEAM_IMC )
+	{
+		tempSetting.shipModel = $"models/vehicle/goblin_dropship/goblin_dropship_hero.mdl"
+		tempSetting.flyinSound = "Goblin_IMC_Evac_Flyin"
+		tempSetting.hoverSound = "Goblin_IMC_Evac_Hover"
+		tempSetting.flyoutSound = "Goblin_IMC_Evac_FlyOut"
+	}
+	if( team == TEAM_MILITIA )
+	{
+		tempSetting.shipModel = $"models/vehicle/crow_dropship/crow_dropship_hero.mdl"
+		tempSetting.flyinSound = "Crow_MCOR_Evac_Flyin"
+		tempSetting.hoverSound = "Crow_MCOR_Evac_Hover"
+		tempSetting.flyoutSound = "Crow_MCOR_Evac_Flyout"
+	}
```

Comment:
Since the function `Evac` can be called by anything, and it's not necessarily an epilogue thing, there should be default behaviour for teams other than `TEAM_MILITIA` and `TEAM_IMC`

Reply:
Simple one line change, `if( team == TEAM_MILITIA )` to `else`